plugins/sql/src/plugin.rs

```diff
@@ -348,7 +348,9 @@ impl Builder {
                     drop(lock);
 
                     app.manage(instances);
-                    app.manage(Migrations(Mutex::new(self.migrations.take().unwrap_or_default())));
+                    app.manage(Migrations(Mutex::new(
+                        self.migrations.take().unwrap_or_default(),
+                    )));
```

Comment:
Ditto, change in other plugin.

Reply:
it's `cargo fmts` fault 🤷🏻‍♂️ 😂

Reply:
And that's why i should stop using the web editor for small changes in rust files :))